README.md

```diff
@@ -563,6 +600,32 @@ spinnaker:
 ```
 
 ## Troubleshooting
+### Kubernetes Resource Handling
+### Discovering Resources
+
+- All kubernetes resources deployed via the plugin are labeled with the label `md.spinnaker.io/plugin: k8s`.
+- Spinnaker's clouddriver labels deployed resources with the name of the application, e.g. for
+  an application named `spinmd`, your created Kubernetes resources for this application are labeled with
+  `app.kubernetes.io/name: spinmd`.
+  
+The combination should allow you to query for application resources created by the plugin.
+
+Example commands:
+```bash
+# Get all resources created by this plugin
+kubectl get all -l md.spinnaker.io/plugin -A
+# Get all resources belongs to application named spinmd
+kubectl get all -l md.spinnaker.io/plugin=k8s,app.kubernetes.io/name=spinmd -A
+# Get jobs created by verifyWith for environment named dev
+kubectl get job -l md.spinnaker.io/verify-environment=dev -A
+```
+
+
+### Cleaning up resources
```

Comment:
probably better to put the section on `cleaning of resources` outside of the troubleshooting section.